src/logic/safe/store/actions/createTransaction.ts

```diff
@@ -143,7 +145,8 @@ const createTransaction = (
   const nonce = await getNewTxNonce(txNonce?.toString(), lastTx, safeInstance)
   const isExecution = await shouldExecuteTransaction(safeInstance, nonce, lastTx)
   const safeVersion = await getCurrentSafeVersion(safeInstance)
-  const safeTxGas = await estimateSafeTxGas(safeInstance, safeAddress, txData, to, valueInWei, operation)
+  const safeTxGas =
+    safeTxGasArg || (await estimateSafeTxGas(safeInstance, safeAddress, txData, to, valueInWei, operation))
```

Comment:
Should we still estimate, to be able to display if the provided safeTxGas seems too low or that the tx is potentially failing

Reply:
We can, but it will require some changes